requirements-dev.txt

```diff
@@ -14,7 +14,6 @@ cffi==1.11.2
 chalice==1.1.0
 chardet==3.0.4
 click==6.7
-clickclick==1.2.2
```

Comment:
… is a dependency of connextion. The next time someone follows the agreed method to bump a dependency or introduce a new one, pip will reintroduce the dependency.